docs/querying/sql-functions.md

```diff
@@ -1357,6 +1357,13 @@ Truncates a numerical expression to a specific number of decimal digits.
 
 Parses `expr` into a `COMPLEX<json>` object. This operator deserializes JSON values when processing them, translating stringified JSON into a nested structure. If the input is not a `VARCHAR` or it is invalid JSON, this function will result in a `NULL` value.
 
+## UNNEST
+
+`UNNEST(source)) as UNNESTED (target)`
```

Comment:
This actually should be `UNNEST(source)) as table_alias_name (column_alias_name)`

Reply:
Here are a couple of things on SQL unnest:

1. The unnest sql function does not remove any duplicates/nulls in an array. Nulls will be treated as any other value in an array. If there are multiple nulls within the array, a record corresponding to each of the null will be created
2. The native unnest has an option of specifying an allowList which cannot be specified through the SQL counterpart
3. Unnest does not work on arrays inside complex JSON types yet.
4. Unnest cannot be used at ingestion time
5. Unnest preserves the ordering in the array which is being unnested
6. Unnest is not supported in MSQ yet (some work needs to be done there)